crates/ruff_python_ast/Cargo.toml

```diff
@@ -13,6 +13,7 @@ ruff_text_size = { workspace = true, features = ["serde"] }
 
 anyhow = { workspace = true }
 bitflags = { workspace = true }
+memchr = "2.5.0"
```

Comment:
Nit: I think we tend to alphabetize here

Reply:
I think I tried to alphabetize... Let me use CLion, it knows the alphabet better than I.